project_templates/stack_package/cookiecutter.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Just to be clear, these are defaults, right?  If so, do you really want to default the package_name?  Won't every package, but necessity have a different name?

Reply:
With Cookiecutter you *have* to provide a default. I think people will know to update it. Or are you saying to call it "my_package"?

Reply:
Good enough.  I didn't realize defaults were mandatory.  I trust your judgement on naming.